Upstream: https://anonscm.debian.org/cgit/collab-maint/mhash.git/tree/debian/patches/0007-autotools-namespace-stomping.patch?id=0a8a6cc7cd11d4f1a7dea8e45f5bb353f9ff5f09

From: "Barak A. Pearlmutter" <bap@debian.org>
Date: Mon, 24 Jun 2013 18:05:04 +0100
Subject: autotools-namespace-stomping

Via: git://pkgs.fedoraproject.org/mhash.git
Taken from Gentoo:
http://mirror.its.uidaho.edu/pub/gentoo-portage/app-crypt/mhash/files/mhash-0.9.9-autotools-namespace-stomping.patch
---
 configure.in                     |  1 +
 include/mutils/config.h.in       | 22 ++++++++++++++++++++++
 include/mutils/mhash_config.h.in | 21 ---------------------
 3 files changed, 23 insertions(+), 21 deletions(-)
 create mode 100644 include/mutils/config.h.in

diff --git a/configure.in b/configure.in
index 17836f9..a667d94 100644
--- a/configure.in
+++ b/configure.in
@@ -6,6 +6,7 @@ AC_CONFIG_SRCDIR([lib/mhash.c])
 AM_INIT_AUTOMAKE
 
 AC_DEFINE([MHASH_VERSION], PROGRAM_VERSION, "MHash Version")
+AC_CONFIG_HEADER([include/mutils/config.h])
 AC_CONFIG_HEADER([include/mutils/mhash_config.h])
 
 
diff --git a/include/mutils/config.h.in b/include/mutils/config.h.in
new file mode 100644
index 0000000..a8db4ac
--- /dev/null
+++ b/include/mutils/config.h.in
@@ -0,0 +1,22 @@
+/* Name of package */
+#undef PACKAGE
+
+/* Define to the address where bug reports for this package should be sent. */
+#undef PACKAGE_BUGREPORT
+
+/* Define to the full name of this package. */
+#undef PACKAGE_NAME
+
+/* Define to the full name and version of this package. */
+#undef PACKAGE_STRING
+
+/* Define to the one symbol short name of this package. */
+#undef PACKAGE_TARNAME
+
+/* Define to the version of this package. */
+#undef PACKAGE_VERSION
+
+/* Version number of package */
+#undef VERSION
+
+
diff --git a/include/mutils/mhash_config.h.in b/include/mutils/mhash_config.h.in
index ed22aa5..9e039e6 100644
--- a/include/mutils/mhash_config.h.in
+++ b/include/mutils/mhash_config.h.in
@@ -181,24 +181,6 @@
 /* Define to 1 if your C compiler doesn't accept -c and -o together. */
 #undef NO_MINUS_C_MINUS_O
 
-/* Name of package */
-#undef PACKAGE
-
-/* Define to the address where bug reports for this package should be sent. */
-#undef PACKAGE_BUGREPORT
-
-/* Define to the full name of this package. */
-#undef PACKAGE_NAME
-
-/* Define to the full name and version of this package. */
-#undef PACKAGE_STRING
-
-/* Define to the one symbol short name of this package. */
-#undef PACKAGE_TARNAME
-
-/* Define to the version of this package. */
-#undef PACKAGE_VERSION
-
 /* Define to 1 if the C compiler supports function prototypes. */
 #undef PROTOTYPES
 
@@ -208,9 +190,6 @@
 /* dmalloc */
 #undef USE_DMALLOC
 
-/* Version number of package */
-#undef VERSION
-
 /* Define if using the dmalloc debugging malloc package */
 #undef WITH_DMALLOC
 
